src/main/kotlin/com/exactpro/th2/rptdataprovider/Extensions.kt

```diff
@@ -255,3 +260,20 @@ fun <T> Flow<T>.chunked(size: Int, duration: Duration): Flow<List<T>> {
         }
     }
 }
+
+
+fun Instant.convertToProto(): Timestamp {
+    return Timestamp.newBuilder()
+        .setNanos(this.nano)
+        .setSeconds(this.epochSecond)
+        .build()
+}
```

Comment:
If I remember correctly, this method exists in common-j. You can use the one from common instead